packages/office-ui-fabric-react/src/components/Spinner/Spinner.tsx

```diff
@@ -1,27 +1,35 @@
 import * as React from 'react';
 import { css } from '../../Utilities';
-import { ISpinnerProps, SpinnerType } from './Spinner.Props';
+import {
+  ISpinnerProps,
+  SpinnerType, // Deprecated and will be removed at >= 2.0.0. Use SpinnerSize instead.
+  SpinnerSize
+} from './Spinner.Props';
 import './Spinner.scss';
 
 export class Spinner extends React.Component<ISpinnerProps, any> {
   public static defaultProps: ISpinnerProps = {
-    type: SpinnerType.normal
+    size: SpinnerSize.medium
   };
 
   public render() {
-    let { type, label, className } = this.props;
+    let { type, size, label, className } = this.props; // TODO remove deprecated type property at >= 2.0.0
 
     return (
       <div className={ css('ms-Spinner', className) }>
         <div className={ css('ms-Spinner-circle',
-          { 'ms-Spinner--normal': type === SpinnerType.normal },
-          { 'ms-Spinner--large': type === SpinnerType.large })
+          { 'ms-Spinner--xSmall': size === SpinnerSize.xSmall },
+          { 'ms-Spinner--small': size === SpinnerSize.small },
+          { 'ms-Spinner--medium': size === SpinnerSize.medium },
+          { 'ms-Spinner--large': size === SpinnerSize.large },
+          { 'ms-Spinner--normal': type === SpinnerType.normal }, // TODO remove deprecated value at >= 2.0.0
```

Comment:
Hmm it is unclear what "normal" would map to. Shouldn't we be mapping normal to one of the non-deprecated values?